features/step_definitions/inspection_steps.rb

```diff
@@ -43,14 +43,14 @@
     count = 0
     while count < attempts
       break if element_exists("* {text CONTAINS[c] '#{text}'}")
-      scroll_down
+      pan_down
```

Comment:
Hey, I updated these to use "scroll" instead of "pan" due to broader support in calabash versions
https://github.com/dimagi/commcare-ui-tests/commit/f4558ff59b321eee56f127d4ba614d6b16ce867b

Is there a specific issue that's occurring with call to scroll that was corrected by switching to pan?